src/release/main.d

```diff
@@ -639,7 +639,7 @@ ActionList prepareMinorRelease ( ref HTTPConnection con, ref Repository repo,
 
                 // Checkout previous branch so we can remove the rel notes
                 list.actions ~= new LocalAction(
-                                 ["git", "checkouts", current_branch.toString],
+                                 ["git", "checkout", current_branch.toString],
```

Comment:
That was probably me... whoops!

How hard would it be to add a test for this so I don't break it again? :grimacing:

Reply:
This is actually covered by the tests added here